templates/spec/components/logo_component_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require "solidus_starter_frontend_helper"
+
+RSpec.describe LogoComponent, type: :component do
+  describe '#call' do
+    let(:logo) { page.find('a') }
+    let(:logo_image) { logo.find('img') }
+
+    before do
+      render_inline(logo_component)
+    end
+
+    context 'by default' do
+      let(:logo_component) { described_class.new }
+
+      it 'renders a link to the root path' do
+        expect(logo[:href]).to eq('/')
+      end
+
+      it 'renders the default logo image' do
+        expect(logo_image[:src]).to match(%r{/assets/logo/solidus-\S+\.svg})
+      end
+    end
+
+    context 'with an image_path' do
+      # For convenience, we're setting the argument to an existing image in the
+      # code base. Otherwise, we'd get an
+      # Sprockets::Rails::Helper::AssetNotFound for the image_path.
+      let(:image_path) { 'icon-delete.svg' }
```

Comment:
I couldn't find a way to stub the image asset. Any ideas?

Reply:
You can set `config.assets.unknown_asset_fallback` to `true` for this test. See https://github.com/rails/sprockets-rails/blob/master/README.md#initializer-options. The exception won't be raised in that case:

https://github.com/rails/sprockets-rails/blob/60f032e8a42e63a2179225392ecdb38f4ac06fa4/lib/sprockets/rails/helper.rb#L84

Otherwise, you could use a fixture image.

Reply:
Thank! @waiting-for-dev ! I'll give it a try :+1:

Reply:
@waiting-for-dev I ended up inlining the helper, so the spec is no longer necessary.